content/python/concepts/iterators/terms/iter/iter.md

```diff
@@ -41,6 +41,7 @@ print(next(cars_iter))
 
 This will print the following output:
 
+THIS NEEDS TO BE UPDATED !
```

Comment:
What is the intent of inserting this line here. It doesn't contribute anything to the current entry, and all caps is at odds with Docs style. Please reach out if you need help on how to do a proper PR to this repo.

Reply:
Hi @SSwiniarski 
It's only a "placeholder" to see where this needs to be updated.
The proposal is not done yet ... I understood you wanted me to make a PR and after that edit it.
Did I misunderstood the request made by @Dusch4593 ?
Shall I directly propose a correction and PR it ?

Reply:
@mcuocolo Generally we want the edits to be done before a PR is submitted. At that point it should be ready to be reviewed. It would probably be best if you withdraw this PR and submit another when you're done with the edits and ready for them to be reviewed.